### DISCUSSION TITLE
Error code 128 stdout

### DISCUSSION THREAD

**Discussion by faidhi066** [atom/atom#23545] [Q&A]
<img width="600" alt="Screenshot 2022-01-31 at 23 34 07" src="https://user-images.githubusercontent.com/66812629/151890386-d41568ea-01ea-4034-a0a6-565f4774b95e.png">

Do you why this happens? It works fine when I use the SSH link.

**Comment by CarlosGordon** [comment]
The same happens to me, but also with the SSH link... so I have no way to clone repositories to Atom... 
Any help would be much appreciated!
